README.md

```diff
@@ -212,22 +212,23 @@ Counted as number of nanoseconds from `1970-01-01T00:00:00.000000000Z`.
 
 ### Constructor
 ```js
-new Instant(milliseconds[, nanoseconds])
+new Instant(nanoseconds)
 ```
 
 #### Parameters
- - `milliseconds` : Integer value representing the number of milliseconds elapsed from 1970-01-01 00:00:00.000 UTC, without regarding leap seconds.
- - `nanoseconds` : Optional. Integer value representing the nanosecond within the millisecond.
+ - `nanoseconds` : BigInt value representing the number of nanoseconds elapsed from 1970-01-01 00:00:00.000 UTC, without regarding leap seconds.
```

Comment:
This PR is about renaming ZonedInstant; unrelated changes to Instant should be removed from it.